### PR TITLE
Adds big csters event

### PR DESCRIPTION
**Pull request by bdamerac**
Updated Big Csters fall 2015 event (balancing life)

### CODE REVIEW

**Comment by evykassirer** [comment]
it's kinda annoying to search through the posts so here is the link

https://wics-site-pr-218.herokuapp.com/2015/10/big-csters-balanced-life-s16/

File: content/F2015/F15-event-bigcsters-balanced-life.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+role for our well-being. We will be sharing some tips and strategies to
+balance your super-busy student life. Come and meet your fellow CSters!
+
+Like previous events, dinner will be provided!
+
+Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
+
+## Event Details ##
+
++ **Who:** [Women and non-binary individuals] (http://wics.uwaterloo.ca/faq).
```
Comment (zarinaaa) [inline content/F2015/F15-event-bigcsters-balanced-life.md]: Remove the space between the link and [Women and non-binary individuals]

**Comment by fboxwala** [comment]
I'm confused, why is this PR up? This event page doesn't look different than the one we already have up.

File: content/F2015/F15-event-bigcsters-balanced-life.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Some of us have had times when we think one aspect of our life is more
+important than the other, such as studying over hanging out with friends
+or being with family. However, we should be aware that everything plays a
+role for our well-being. We will be sharing some tips and strategies to
+balance your super-busy student life. Come and meet your fellow CSters!
+
+Like previous events, dinner will be provided!
+
+Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
```
Comment (fboxwala) [inline content/F2015/F15-event-bigcsters-balanced-life.md:17]: It's better practice to make these links relative, like this:

`[women-only]({filename}/pages/faq.md)`

File: content/F2015/F15-event-bigcsters-balanced-life.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Event Details ##
+
++ **Who:** [Women and non-binary individuals](http://wics.uwaterloo.ca/faq).
```
Comment (fboxwala) [inline content/F2015/F15-event-bigcsters-balanced-life.md:21]: Same thing

File: content/F2015/F15-event-bigcsters-balanced-life.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
++ **What:** Balancing Lifestyle
++ **Where:** MC 5479
++ **When:** Wednesday, October 21, 5:30 - 7 PM
```
Comment (fboxwala) [inline content/F2015/F15-event-bigcsters-balanced-life.md:24]: Can you make it like this:
`Wednesday Oct. 21, 5:30&ndash;7:00PM`
This makes it pretty and consistent with other posts :)

Comment (fboxwala) [inline content/F2015/F15-event-bigcsters-balanced-life.md]: Can you make it like this:
`5:30&ndash;7:00PM`
Not sure how much the &ndash; matters, but this makes it pretty and consistent with other posts :)

**Comment by fboxwala** [comment]
So you've fixed the typo in the name of the PR but not in the actual commit message. You're going to have to do a `git rebase -i` to fix this, and squash the next commit down into this one. 
On squashing commits:

https://github.com/wics-uw/git-workshop-W16/wiki/Squashing-Commits
Ping me if you need help!

Also, you should really be making pull requests from a `featurebranch` and not your fork's `master` branch. That way if something goes wrong your master branch is still okay. (I did the same thing by accident at other day at work)
In general make sure you're following the workflow described in the README of our repo/the steps in the wiki:

https://github.com/wics-uw/git-workshop-W16/wiki/Part-3:-Feature-Development

**Comment by evykassirer** [comment]
@Bhargavi1997 how're the changes going? You still need to fix the link and dash and squash the commits.

**Comment by bdamerac** [comment]
@evykassirer I did most of the changes and then it struck me that absolute vs relative links is an ongoing issue so I decided to wait until it takes a form.

**Comment by evykassirer** [comment]
it's no longer an ongoing issue - we need to change them (See Elana's comment in the issue about it)

so you can make it relative here

**Comment by bdamerac** [comment]
Okay! I'll complete it by 9pm today!

**Comment by evykassirer** [comment]
sweet! thanks for the quick responses 👍

**Comment by bdamerac** [comment]
I created a pull request from a different branch and not from master this time.

**Comment by evykassirer** [comment]
sweet! thanks for doing that

**Pull request closed without merging**